multichain-on-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,213 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "storageAccountNamePrefix": {
+      "type": "string",
+      "maxLength": 11,
+      "metadata": {
+        "description": "Name prefix of the Storage Account"
+      }
+    },
+    "vmDnsName": {
```

Comment:
Hi,

Would be great if you could mod it after the merge as I am currently away
from my computer. Thanks.

Simon

On Wednesday, January 20, 2016, Kay Singh <notifications@github.com> wrote:

> In multichain-on-ubuntu/azuredeploy.json
> <https://github.com/Azure/azure-quickstart-templates/pull/1120#discussion_r50321670>
> :
>
> > @@ -0,0 +1,213 @@
> > +{
> > +  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
> > +  "contentVersion": "1.0.0.0",
> > +  "parameters": {
> > +    "storageAccountNamePrefix": {
> > +      "type": "string",
> > +      "maxLength": 11,
> > +      "metadata": {
> > +        "description": "Name prefix of the Storage Account"
> > +      }
> > +    },
> > +    "vmDnsName": {
>
> Yes please. It's good to merge all templates with consistency. Shouldn't
> be a big change since the template deploys successfully already.
>
> Alternatively, since this is a small change, I can dot it after the merge
> as well if don't mind.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Azure/azure-quickstart-templates/pull/1120/files#r50321670>
> .
>